### ISSUE TITLE
missing include for std::end

### ISSUE THREAD

**Issue by Trass3r** [dji-sdk/Onboard-SDK#132]
```diff
diff --git a/osdk-core/platform/linux/src/linux_serial_device.cpp b/osdk-core/platform/linux/src/linux_serial_device.cpp
index 755ca88..9c92c74 100644
--- a/osdk-core/platform/linux/src/linux_serial_device.cpp
+++ b/osdk-core/platform/linux/src/linux_serial_device.cpp
@@ -14,6 +14,8 @@
 
 #include "linux_serial_device.hpp"
 #include <algorithm>
+#include <iterator>
+
 using namespace DJI::OSDK;
 
 /*! Implementing inherited functions from abstract class DJI_HardDriver */
```

**Comment by rohitsantdji** [comment]
Could you provide your platform and what version of gcc you are using?

**Comment by Trass3r** [comment]
Ubuntu 17.04 with gcc 6.3

**Comment by rohitsantdji** [comment]
Thanks! We don't yet test on Ubuntu 17.04. I'll push a fix soon.

**Comment by rohitsantdji** [comment]
Addressed in e6c293341fb5461e8b364a584ca2bae8855cb5d5
